packages/devtools_app/lib/src/config_specific/launch_url/_launch_url_web.dart

```diff
@@ -0,0 +1,11 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be found
+// in the LICENSE file.
+
+import 'dart:html';
+
+final isVSCodeEmbedded = window.navigator.userAgent.contains('Electron');
+
+void launchUrlVSCode(String url) {
+  window.parent?.postMessage({'command': 'launchUrl', 'data': url}, '*');
```

Comment:
To mirror my comment in the Dart-Code repo, I think it's better if this is `'data': { 'url': url }` so that if we need to pass additional data through in future it's not a breaking change (from a string to a map/object).

Reply:
Done